### DISCUSSION TITLE
Ps1 games not getting vibration when playing on my ps2 using BLUERETRO receiver

### DISCUSSION THREAD

**Discussion by Espinete7** [darthcloud/BlueRetro#1083] [General]
Hi,

I have one problem with the BLUERETRO receiver, I can’t find info about it.
I play ps1 and ps2 games in my ps2 original hardware. When I play with a wired dualshock 2 vibration works flawlessly, no matter if I use the discs or if I use pops starter for ps1, vibration works.
When I use the BLUERETRO receiver, with ps2 games I got the vibration, but when I play my ps1 games I don’t have any vibration in any of the games I tried.
What can I do to have vibration playing my ps1 games in my ps2?

Thanks in advance!

**Comment by trunk70** [comment]
Sam problem with PS1 Games on Mister FPGA, no vibration at all in PS1 games with Blueretro, works fine with wired original gamepad (Snac, GP advance etc...), i don't understand...
